grassroots-frontend/src/routes/PhoneCanvass/Create.tsx

```diff
@@ -50,6 +55,7 @@ function CreatePhoneCanvass(): JSX.Element {
           body: {
             csv: csvText,
             name: phoneCanvass.name,
+            // Note: audio field not yet sent to backend (will be implemented in PR #3)
```

Comment:
We don't have any examples of this in the backend right now. It might be a bit gnarly, I'm not sure.

Docs are here:
https://docs.nestjs.com/techniques/file-upload